charts/pega/KafkaClusterRequirement.md

```diff
@@ -46,7 +46,7 @@ Principal |Resource Type  | Resource Name     | Operation | Permission Type | Pa
 ---         |---         | ---     | ---    | ---        | ---
 User:\<user-name\> | TOPIC       | \<Prefix\> as in 'stream.streamNamePattern' | ALL    | ALLOW      | PREFIXED
```

Comment:
I don't think these details are clear at this point would this be more clear for 47 + 49??
\<Prefix\> for example, \<'stream.streamNamePattern'\>

or is it
\<Prefix\> for example, for the `pega-stream` prefix, use the string '\<pega-stream\>.streamNamePattern'

Reply:
@girirajguptagithub can we give examples for how you use the Resource Name string derived from the Pega parameters in a given Kafka acl command like this one that I copies from the Kafka doc??>
> bin/kafka-acls.sh --authorizer-properties zookeeper.connect=localhost:2181 --add --allow-principal User:Peter --allow-host 198.51.200.1 --producer --topic '*'

I think an example would greatly help clients to understand how to integrate our stream.streamNamePattern prefix into these precise Kafka commands...just saying that an example would be really useful
Show us how you'd add a group or a

Reply:
@taz-pega-work added an example.